package.json

```diff
@@ -49,21 +49,22 @@
   },
   "homepage": "https://github.com/fletcherist/yandex-dialogs-sdk#readme",
   "dependencies": {
-    "@types/node-fetch": "^2.1.2",
     "node-fetch": "^2.1.2"
   },
   "devDependencies": {
-    "@types/express": "^4.16.0",
+    "@types/debug": "^0.0.30",
+    "@types/node-fetch": "^2.1.2",
     "@types/jest": "^23.1.3",
     "@types/node": "^10.5.1",
     "conventional-changelog-cli": "^2.0.0",
     "conventional-github-releaser": "^3.1.0",
     "husky": "^0.14.3",
+    "debug": "^3.1.0",
     "jest": "^23.2.0",
     "prettier": "^1.13.7",
     "ts-jest": "^23.0.0",
     "tslint": "^5.10.0",
-    "typescript": "^2.9.2",
+    "typescript": "^3.0.1",
```

Comment:
Please move typescript update to separate PR. At least to separate commit. Commit message does not reflect changes made in this commit.